### PR TITLE
Remove trailing forward slash from BRH aggMDS

### PR DESCRIPTION
**Pull request by jawadqur**
Link to Jira ticket if there is one:

### Environments
brh.data-commons.org
JCOIN

### Description of changes
Remove trailing slash after JCOIN url in BRH agg MDS

### CODE REVIEW

**Review by mfshao — DISMISSED** [review]

File: brh.data-commons.org/metadata/aggregate_config.json [hunk 1]
```diff
@@ -43,8 +43,8 @@
       }
     },
     "JCOIN": {
-      "mds_url": "https://jcoin.datacommons.io/",
-      "commons_url": "jcoin.datacommons.io/",
+      "mds_url": "https://jcoin.datacommons.io",
```
Comment (mfshao) [inline brh.data-commons.org/metadata/aggregate_config.json:46]: oh actually I think you should not change this

Comment (nss10) [reply brh.data-commons.org/metadata/aggregate_config.json:46]: Do you mean the `mds_url` @mfshao ?

File: brh.data-commons.org/metadata/aggregate_config.json [hunk 2]
```diff
@@ -43,8 +43,8 @@
...
+      "commons_url": "jcoin.datacommons.io",
```
Comment (nss10) [inline brh.data-commons.org/metadata/aggregate_config.json:47]: There is one more place in this file which has a trailing `/` (`Cancer Imaging Data Commons`). I wonder why do we have certain urls with a trailing `/` and a few urls which don't?

* Could this have been a typo?
* If yes can we fix both of these under a single PR?

**Review by nss10 — CHANGES_REQUESTED** [review]

File: brh.data-commons.org/metadata/aggregate_config.json (hunk 1 above)
Comment (jawadqur) [reply brh.data-commons.org/metadata/aggregate_config.json:46]: I matched whatever all the other `mds_url` was set to, so I think we should keep it like this.

File: brh.data-commons.org/metadata/aggregate_config.json (hunk 2 above)
Comment (jawadqur) [reply brh.data-commons.org/metadata/aggregate_config.json:47]: Good catch, let me push another commit to address that.

**Review by mfshao — APPROVED** [review]

**Review by nss10 — APPROVED** [review]

**Pull request merged**